build.gradle

```diff
@@ -1,37 +1,63 @@
-//------------------------------------------- PLUGIN DEFINITION -----------------------------------------------//
-
-plugins {
-    id "java-gradle-plugin"
-    id "eclipse"
-    id "jacoco"
+/**
+Template file for classes using GradleRIO
+@author Jaci
+*/
+
+
+buildscript {
+  repositories {
+    //mavenLocal()
+	mavenCentral()
```

Comment:
I would say it's more for us than them, as long as there's internet it's not an issue